pypwrctrl/pypwrctrl.py

```diff
@@ -116,12 +117,23 @@ def __init__(self, pin=77, pout=75, user="admin", password="anel",
 
         self.iface = iface
         self.devices = []
+        retry = 5
 
         self.sin = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
         if self.iface is not None:
                 self.sin.setsockopt(socket.SOL_SOCKET, socket.SO_BINDTODEVICE,
                                     bytes(iface, 'UTF-8'))
-        self.sin.bind(('0.0.0.0', pin))
+        while (retry >= 1):
+            try:
+                self.sin.bind(('0.0.0.0', pin))
+                break
+            except OSError as err:
+                if retry == 1:
+                    raise err
+                print("Could not open listening port, going to retry.", file=sys.stderr)
```

Comment:
This hides "PermissionError: [Errno 13] Permission denied" which is common because port 77 is privileged and default